src/prebid.js

```diff
@@ -334,12 +305,8 @@ $$PREBID_GLOBAL$$.removeAdUnit = function (adUnitCode) {
 };
 
 $$PREBID_GLOBAL$$.clearAuction = function() {
```

Comment:
Removed this method. It will be deprecated in 1.0 since now the only purpose of this method remains is to call syncCookies which now i have moved when auction ends